sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/ModelConverter.java

```diff
@@ -0,0 +1,39 @@
+package com.azure.digitaltwins.core.models;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+
+/**
+ * Creates publicly facing object types
+ */
+public class ModelConverter {
+    public static DigitalTwinModelData convertToDigitalTwinModelData(
+        String id,
+        Object model,
+        OffsetDateTime uploadTime,
+        Map<String, String> description,
+        Map<String, String> displayName,
+        Boolean isDecommissioned)
```

Comment:
Why not have the converted take in the PL `ModelData` directly?

Reply:
good question, we cannot expose any models from the implementation package to the user. and now that we are hiding the ModelData, we can't take it as a parameter here since this is in the core.model package